siliconcompiler/tools/vpr/vpr.py

```diff
@@ -76,6 +79,10 @@ def runtime_options(chip):
 
     options = []
 
+    device_code = chip.get('fpga', part_name, 'var', 'device_code')
+
+    options.append(f"--device {device_code[0]}")
```

Comment:
make this a helper function so genfasm can also get these arguments

Reply:
bitstream.py current gets these by calling `vpr.runtime_options()` -- is there any benefit to restructuring genfasm's tool driver such that there is a genfasm.py `runtime_options()` that does this instead, and then having bitstream.py call genfasm.py's `runtime_options()`?